package.json

```diff
@@ -54,6 +54,7 @@
     "react-dom": ">=15.0.0"
   },
   "dependencies": {
+    "create-react-context": "^0.2.3",
```

Comment:
i don't really want to depend on CRC, it's a biggish polyfill and folks don't like the license

Reply:
i'm not sure if it's worth doing a major bump yet for this either :/